phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexMetadataIT.java

```diff
@@ -100,7 +100,7 @@ private static void assertActiveIndex(Connection conn, String schemaName, String
         String fullTableName = SchemaUtil.getTableName(schemaName, tableName);
         conn.createStatement().executeQuery("SELECT count(*) FROM " + fullTableName).next(); // client side cache will update
         PhoenixConnection pconn = conn.unwrap(PhoenixConnection.class);
-        pconn.getMetaDataCache().getTable(new PTableKey(pconn.getTenantId(), fullTableName)).getIndexMaintainers(ptr, pconn);
```

Comment:
FYI, I made this change to master, so if you could rebase the txn branch, these diffs should go away.